test/e2e/externalconnectivity_test.go

```diff
@@ -0,0 +1,72 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"testing"
+
+	pkgTest "github.com/knative/pkg/test"
+	"github.com/knative/serving/test"
+)
+
+func TestExternalConnectivity(t *testing.T) {
+	t.Parallel()
+	clients := Setup(t)
+
+	names := test.ResourceNames{
+		Service: test.ObjectNameForTest(t),
+		Image:   "externalconnectivity",
```

Comment:
Could we use the `httpproxy` image we already have as a test image with a TARGET_HOST set to the outside world? We could then fire a request against that proxy and get the response from google.com directly to do our checks with.

Reply:
+1 

The number of test images we have continues to grow, and `httpproxy` seems like a great fit here to re-use without adding a new image.